specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Clusters.json

```diff
@@ -0,0 +1,501 @@
+          "description": "The associated key properties."
+        }
+      },
+      "description": "Log Analytics cluster patch properties."
+    },
+    "ClusterPatch": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterPatchProperties",
+          "description": "Log Analytics cluster properties."
+        },
+        "sku": {
+          "$ref": "#/definitions/sku",
+          "description": "The sku properties."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "The top level Log Analytics cluster resource container.",
+      "x-ms-azure-resource": true
+    },
+    "Cluster": {
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
+        },
+        "sku": {
+          "$ref": "#/definitions/sku",
+          "description": "The sku properties."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterProperties",
+          "description": "Log Analytics cluster properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "The top level Log Analytics cluster resource container.",
+      "x-ms-azure-resource": true
+    },
+    "ClusterListResult": {
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          },
+          "description": "A list of Log Analytics clusters."
+        }
+      },
+      "description": "The list clusters operation response."
+    },
+    "keyVaultProperties": {
+      "properties": {
+        "keyVaultUri": {
+          "description": "The Key Vault uri which holds they key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The version of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        }
+      }
+    },
+    "sku": {
+      "properties": {
+        "capacity": {
+          "description": "The capacity value",
+          "type": "integer",
+          "format": "int64"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the SKU.",
+          "enum": [
```

Comment:
this defined differently from the other one which blocked dotnet SDK generation

Reply:
sure